tests/providers/cncf/kubernetes/operators/test_pod.py

```diff
@@ -327,7 +327,7 @@ def test_find_pod_labels(self):
         self.run_pod(k)
         _, kwargs = k.client.list_namespaced_pod.call_args
         assert kwargs["label_selector"] == (
-            "dag_id=dag,kubernetes_pod_operator=True,run_id=test,task_id=task,"
+            "dag_id=dag,foo=bar,kubernetes_pod_operator=True,run_id=test,task_id=task,"
```

Comment:
Rather can we have a different test case that searches by label params and leave this test case, as-is?

Reply:
Thank you for comment and guide..!

But, I think.. it will be ambiguity between two test cases, when new test case is added for this.. isn't it..?
- (existing) Test Case 1: test for making label selector logic
- (new) Test Case 2: test for ???...

Reply:
Test case 2 should only be searching by `foo=bar` which is one of the changes introduced in this PR.

Reply:
Sure, I added the new test case following your guide :)